src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.AliasesAndUsings.cs

```diff
@@ -85,14 +84,41 @@ private SingleNamespaceDeclaration GetMatchingNamespaceDeclaration(CSharpSyntaxN
             throw ExceptionUtilities.Unreachable();
         }
 
-        private static AliasesAndUsings GetOrCreateAliasAndUsings(
+        private AliasesAndUsings GetOrCreateAliasAndUsings(
             ref ImmutableSegmentedDictionary<SingleNamespaceDeclaration, AliasesAndUsings> dictionary,
             SingleNamespaceDeclaration declaration)
         {
-            return RoslynImmutableInterlocked.GetOrAdd(
-                ref dictionary,
-                declaration,
-                static _ => new AliasesAndUsings());
+            if (dictionary.IsDefault)
```

Comment:
❓ Is this method ever called in release mode for cases where `declaration` doesn't have any aliases, global usings, or usings (the three conditions checked on line 100)?